### ISSUE TITLE
Website Observations

### ISSUE THREAD

**Issue by coderdiaz** [coderdiaz/v3#1]
## Observations
- [x] **[Performance]** Site is loading only the `dev` CDN `vue` script this isn't correctly in `production` mode.
- [x] **[Accessibility]** <html> element does not have a [lang] attribute.
- [x] **[Best Practices]** Document mus contain a `doctype`.
- [x] **[SEO]** Document does not have a meta description.
- [ ] **[Performance]** Eliminate render-blocking resources.
- [ ] **[Performance]** Defer offscreen images using Lazyload.
- [ ] **[Performance]** Properly size images using `<picture>` element for save cellular data.
- [ ] **[Best Practices]** Displays Images With Incorrect Aspect Ratio.
- [x] **[Performance]** Defer unused CSS for Font Awesome.
- [x] **[Performance]** Use next-generation image format.
